### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by elasticdotventures**
Hi @3DLIRIOUS , I've forked your repo.

I'm going to take an attempt at building some new tooling (build scripts, validation tests, github badges) in RUST cargo and Github actions (related to documentation, building install packages for openSCAD). 
Maybe 3rd contributor is the charm ?  I'm keen to collaborate if you are still interested in this project. 

Planning to roll it into my own project:
https://github.com/elasticdotventures/awesome-openscad

The idea is to have a unified object & plugin installer for the openscad awesome list.   Presently trying to bring all the peoples objects & work together in git is a bit complicated and I'm hoping to address (at least the blender portion, and will try to integrate the vscode LSP [language service provider]) into the blender interface to allow syntax checking.

### CODE REVIEW

**Comment by elasticdotventures** [comment]
TLDR: Please Sunset or enable Issues in you repo to indicate you are actively working on it. 

I'd also like to say on behalf of the openscad awesome list, first THANKS for what you built.

IF you are finished with this project for now then please update your README indicating the project has been sunset.  You can always have another future sunrise by pulling my/our awesome upstream - AND by sunsetting your project and linking to the preferred upstream version you help the community. 

IF you plan to continue working on this fork, please consider enabling the "issues" feature in github.   The github issues feature was not available when the [](miguelitoelgrande/BlenderSCAD) repo was originally created. 
Not having issues makes non-git savvy users leaving feedback is a bit more difficult & cumbersome,  I think this is especially important in the openscad community (many, perhaps most of the users are designers first, git savvy-coders second)

without issues, it's really difficult to see feedback others gave is impossible  .. the only way to send feedback is a pull request like this, which is a lot of work to ask a question or leave a comment.  Looking forward to a dialog if you are keen. 

Thanks!
